utest/test/keywords/test_keyword_arguments_browsermanagement.py

```diff
@@ -31,3 +31,8 @@ def test_open_browser(self):
         alias = self.brorser.open_browser(url, alias='None',
                                           remote_url=remote_url)
         self.assertEqual(alias, None)
+
```

Comment:
Please write a new test, because this is testing totally different thing than the original test.